infrastructure/nomad/playbooks/templates/jobs/mev-commit-oracle.nomad.j2

```diff
@@ -264,9 +264,11 @@ job "{{ job.name }}" {
             sleep 3
           fi
 
+          {% endraw %}
+          export MEV_ORACLE_RELAY_URLS="{{ job.env['oracle_relay_urls'] }}"
```

Comment:
Is this meant to replace the `MEV_ORACLE_RELAY_URLS` on line 253 above?

Reply:
Yes, when it is set, it will overwrite the value.